### ISSUE TITLE
RF-DETR not predicting more than 100 objects.

### ISSUE THREAD

**Issue by emerson-w** [roboflow/rf-detr#200]
I am currently training RF-DETR with an objective that predicts more than 100 objects. I have set num_queries to 700 as shown below, but the trained model will not predict more than 100 objects. 

model = RFDETRBase()

model.train(
    dataset_dir= ...,
    num_queries = 700,
)

detections = model.predict(image, threshold=0.5)

Any help or advice appreciated.

**Comment by SkalskiP** [comment]
Hi @emerson-w 👋🏻 Thanks so much for your interest in RF-DETR!

This issue was addressed in https://github.com/roboflow/rf-detr/pull/123, but the fix hasn’t been released on PyPI yet. To take advantage of the update, please install RF-DETR directly from source using:

```bash
pip install git+https://github.com/roboflow/rf-detr.git
```

With this version, you'll be able to control the number of detections using the `num_queries` and `num_select` parameters.

I'm closing this issue for now. Feel free to reopen it if you have any further questions or run into additional issues.
